config/connect-avro-distributed.properties

```diff
@@ -57,7 +57,26 @@ internal.value.converter=org.apache.kafka.connect.json.JsonConverter
 internal.key.converter.schemas.enable=false
 internal.value.converter.schemas.enable=false
 
-# Confuent Control Center Integration -- uncomment these lines to enable Kafka client interceptors
+# Confluent Control Center Integration -- uncomment these lines to enable Kafka client interceptors
 # that will report audit data that can be displayed and analyzed in Confluent Control Center
 # producer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringProducerInterceptor
 # consumer.interceptor.classes=io.confluent.monitoring.clients.interceptor.MonitoringConsumerInterceptor
+
+# These are provided to inform the user about the presence of the REST host and port configs
```

Comment:
I'm happy if we just generally improve the configs, but given this is already consistent I wouldn't insist on it.